core/src/avm2/globals/flash/net/net_stream.rs

```diff
@@ -80,3 +82,37 @@ pub fn toggle_pause<'gc>(
 
     Ok(Value::Undefined)
 }
+
+pub fn get_client<'gc>(
+    _activation: &mut Activation<'_, 'gc>,
+    this: Object<'gc>,
+    _args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    if let Some(ns) = this.as_netstream() {
+        return Ok(ns.client().expect("NetStream client should be set").into());
+    }
+
+    Ok(Value::Undefined)
+}
+
+pub fn set_client<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    this: Object<'gc>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let client = args.get_value(0).as_object();
```

Comment:
Not `get_object` for proper error handling?

Reply:
I've tested this and `null`, `undefined`, `3`, and `true` all throw the same Error # 2004. (meanwhile, `MovieClip` and `{}` work fine). Note that this is a setter function, not exactly a "do operation" function.

Reply:
I've added the test demonstrating this.